backend/monitoring/dashboard.py

```
@@ -7,9 +7,62 @@
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+
+# Import proper admin authentication
+try:
+    from deps import get_current_admin_dependency
+except ImportError:
+    # If deps module is not available, create a secure fallback
+    from fastapi import Security
+    
+    security = HTTPBearer()
+    
+    async def get_current_admin_dependency(credentials: HTTPAuthorizationCredentials = Security(security)):
```

Ruff (0.12.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)